src/SlmLocale/Strategy/UriPathStrategy.php

```diff
@@ -74,6 +76,9 @@ public function setOptions(array $options = [])
         if (array_key_exists('redirect_to_canonical', $options)) {
             $this->redirect_to_canonical = (bool) $options['redirect_to_canonical'];
         }
+        if (array_key_exists('default', $options)) {
```

Comment:
I am not sure about one more default locale we already have one maybe we could use an old one instead of this or this is not possible here?

Reply:
i dont know how to access the default locale, so i added a new one just for the strategy. thats also the key to enable the feature